CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ## Upcoming Release
 * Web App
   * Fix internet radio search on IOS app
+* System
+  * Rename Updater to Updater/Admin Control Panel
+  * Add "Admin Settings" tab to the Updater/Admin Control Panel
+    * Add toggleable option to persist system logs to Admin Settings
+    * Move "Set password" page to be accessible via Admin Settings tab
```

Comment:
Unusual formatting as a few of these bullets fall under the umbrella of another, I can untab the latter two entries no problem if we dislike this